data/ships.txt

```diff
@@ -171,30 +171,30 @@ ship "Bactrian"
 	attributes
 		category "Heavy Warship"
 		"cost" 17600000
-		"shields" 17500
-		"hull" 8600
+		"shields" 12400
+		"hull" 9800
 		"required crew" 70
```

Comment:
Right. But I would keep using the Bactrian despite a higher crew salary; it's just that I would be less inclined to play it especially as it becomes more and more uncompetitive at higher tiers. It's just that currently, even though it's a Tier 1 ship, it's still arguably _the most_ competitive ship at higher tiers. Your other balances should fix that. But raising the crew count to _just_ below a Cruiser makes sense—it's going to be expensive to operate, and the expense becomes less and less justified the higher up we go. I think with this crew change I would immediately switch to a Tempest or even a Shield Beetle. It's operation is no longer justified at the higher tiers, and players will find the Bactrian's cost-effectiveness decrease. Just my two cents, and we can also see what MZ thinks.